backend/tests/test_endpoints.py

```diff
@@ -101,3 +122,65 @@ def test_google_login(test_client, db, mocker):
     assert user_data["email"] == "test@rice.edu"
     assert user_data["is_manager"] is False
     assert user_data["is_admin"] is False
+
+
+def test_get_all_users(test_client, db, setup_admin_session):
+    db_user = UserAccount(email="test@rice.edu", is_manager=False, is_admin=False)
+    db_user2 = UserAccount(email="admin@rice.edu", is_manager=False, is_admin=True)
+    db.add(db_user)
+    db.add(db_user2)
+    db.commit()
+    response = test_client.get("/user")
+    data = response.json()
+    assert response.status_code == 200
+    assert len(data) == 2
```

Comment:
It might be nice to validate the shape of at least one of the return objects (this applies to all of the tests below as well). For example, right now if you modified `GetUsersResponse` to where it didn't include `id`, FastAPI will remove it but nothing will fail in our tests. For values like `id` where we may not know / care about the actual value, we can assert something like `assert data[0].get("id") is not None`.